src/components/course/data/hooks.jsx

```diff
@@ -694,16 +698,15 @@ export const useUserSubsidyApplicableToCourse = ({
 
     let applicableUserSubsidy;
 
-    // if course can be redeemed with EMET system, return
-    // the redeemable subsidy access policy.
-    if (isPolicyRedemptionEnabled && redeemableSubsidyAccessPolicy) {
+    // if course can be redeemed with a subsidy access policy, return it as the subsidyType.
+    if (isPolicyRedemptionEnabled) {
```

Comment:
[context] `redeemableSubsidyAccessPolicy` must be true if `isPolicyRedemptionEnabled` is true, so it's unnecessary to check on `redeemableSubsidyAccessPolicy` here again.